strategy-service.ts

```diff
@@ -0,0 +1,5 @@
+import { strategyType, type Candle } from "./types/common";
+
+function strategyService(candle: Candle): strategyType {
```

Comment:
This service doesn't accept a candle. It accepts a record of pairs mapped to an array of candles.
In addition, it returns a record of pairs mapped to a `PositionType`. (This may change in the future, though.)  
Look at the strategy interface.